test/univariate/continuous/noncentralchisq.jl

```diff
@@ -1 +1,12 @@
-test_cgf(NoncentralChisq(3,2), (0.49, -1, -100, -1f6))
+using Distributions
+using Random
+using Test
+
+@testset "Native RNG" begin
+    rng = MersenneTwister(1234)
+    n = 5000
+    d = NoncentralChisq(3, 2)
+    samples = rand(rng, d, n)
+    @test mean(samples) ≈ mean(d) atol=0.05*mean(d)
+    @test var(samples) ≈ var(d) atol=0.1*var(d)
+end
```

Comment:
We should just use the existing test functionality, by making sure these distributions are included in the tests (also with `rand`) run by `test/univariates.jl`.

Reply:
Would something like this work? It more or less tests the "quality" of RNGs.
        ``samples = rand(rng, d, n)
        cdf_v = [cdf(d, x) for x in samples]
        quantiles = [quantile(d, p) for p in cdf_v]
        @test all(quantiles .≈ samples)`` where d is ContinuousUnivariateDistribution

Reply:
No, my point is that we should not come up with such new tests. We should just use the setup in test/univariates.jl, or if that's not possible (eg if mean and variance are not defined) we should at least use https://github.com/JuliaStats/Distributions.jl/blob/ec68da3a8d4a4776367f2d7ca5ec2d4666e29c78/test/testutils.jl#L99.

Reply:
Ah ok. Similar to `test_cgf`.

Reply:
> We should just use the existing test functionality, by making sure these distributions are included in the tests (also with `rand`) run by `test/univariates.jl`.

univariates.jl does not have any method with the `rng` parameter.

Reply:
> No, my point is that we should not come up with such new tests. We should just use the setup in test/univariates.jl, or if that's not possible (eg if mean and variance are not defined) we should at least use
> 
> https://github.com/JuliaStats/Distributions.jl/blob/ec68da3a8d4a4776367f2d7ca5ec2d4666e29c78/test/testutils.jl#L99
> 
> .

Ah, like the `test_cgf` added in some test files?